README.md

```diff
@@ -11,13 +11,13 @@ The book is expected to be helpful for both beginner and experienced Go programm
 Run
 
 ```
-$ git clone gitlab.com/go101/go101
+$ git clone https://gitlab.com/Go101/go101.git
 ```
```

Comment:
@alyjee Thank you very much for this fix!

It was caused I didn't change the old `go get gitlab.com/go101/go101` line fully.

Although it is not very necessary, for consistency, could you modify it more, by using all lower-case letters in the url?